OpenRA.Mods.Common/SpriteLoaders/PngSheetLoader.cs

```diff
@@ -69,24 +66,29 @@ public bool TryParseSprite(Stream s, out ISpriteFrame[] frames, out TypeDictiona
 			{
 				var frameStart = frameRegions[i].X + frameRegions[i].Y * png.Width;
```

Comment:
I think these should be multiplied by `pixelLength` to get the offset for the frame:
```csharp
var frameStart = frameRegions[i].X * pixelLength + frameRegions[i].Y * pixelLength * png.Width;
```